cupy/testing/helper.py

```diff
@@ -563,11 +563,15 @@ def test_func(self, *args, **kw):
     return decorator
 
 
-def numpy_cupy_raises(name='xp', sp_name=None, scipy_name=None,
-                      accept_error=Exception):
+def numpy_cupy_raises(
+        accept_error, *, name='xp', sp_name=None, scipy_name=None):
     """Decorator that checks the NumPy and CuPy throw same errors.
 
     Args:
+         accept_error(Exception or tuple of Exception): Specify
+             acceptable errors. When both NumPy test and CuPy test raises the
+             same type of errors, and the type of the errors is specified with
+             this argument, the errors are ignored and not raised.
```

Comment:
The test with this decorator does not pass when both NumPy test and CuPy test don't raise any error.

Reply:
I agree the wording could be improved to be more unambiguous, but it's not the point of this PR.